privacy-snapshot.json

```diff
@@ -18,6 +18,7 @@
   "fonts.gstatic.com",
   "gas-api.metaswap.codefi.network",
   "github.com",
+  "goerli.infura.io",
```

Comment:
Is this addition intensional?

Reply:
It's not intentional in that we know exactly why it suddenly started hitting goerli.infura.io, but it is intentional that it fails without this.